uiowa/uiowa
# Hotfix: change ddev webserver to apache.

Followup from #4458.

# Problem
We're not using apache locally and should be b/c thats what Acquia uses.

# How to test
- Go to http://default.uiowa.ddev.site and see that you are not redirected to https because nginx doesn't use .htaccess.
- Check out this branch and `ddev restart`.
- Go to that URL again.
- ??
- Profit
- Test other stuff.
